Presentation.Web/app/helpers/kendo-overview-helper.ts.ts

```diff
@@ -0,0 +1,30 @@
+﻿module Kitos.Helpers {
+
+    export class KendoOverviewHelper {
+
+        /**
+          *  creates a select dropdown for given dataSource
+          * @param args - should contain kenoGrid elements
+          * @param dataSource - dataSource of the dropdown, should contain 'id' and 'text' properties
```

Comment:
in stead of writing it in text, create an interface type and replace `any`with that in the argument list